themes/bootstrap3/templates/RecordDriver/DefaultRecord/result-list.phtml

```diff
@@ -146,7 +146,7 @@
           if(!$this->driver->isCollection()):
             foreach ($urls as $current): ?>
               <a href="<?=$this->escapeHtmlAttr($this->proxyUrl($current['url']))?>" class="fulltext" target="new">
-                <?=$this->icon('external-link') ?> <?=($current['url'] == $current['desc']) ? $this->transEsc('Get full text') : $this->escapeHtml($current['desc'])?></a><br/>
+                <?=$this->icon('external-link', 'icon-link__icon') ?> <?=($current['url'] == $current['desc']) ? $this->transEsc('Get full text') : $this->escapeHtml($current['desc'])?></a><br/>
```

Comment:
Does the link text need a span around it? The underline looks a little too big here:
![image](https://user-images.githubusercontent.com/309069/197259733-9924b608-9b37-4d0d-82b9-fcb81e65d2a1.png)
Maybe this is also an opportunity to put text and icon on separate lines in the template to improve readability.